### DISCUSSION TITLE
In addition to jsx attr configuration, add object key config

### DISCUSSION THREAD

**Discussion by coreylight** [lingui/eslint-plugin#17] [Ideas]
We have the `ignoreAttribute` option which is great and highly needed.
I have written codemods to automatically wrap strings in lingui components, and in my codemod I additionally check against a whitelist of object keys such as:
```ts
const test = {
  id: 'This is not localized'
  text: 'This is'
}
```
where `id` is opted out of the no-unlocalized lint rule
This is very similar to jsx attrs.
What do we think about including extra code to allow this config? Or apply the attrs config to objects too?

**Comment by timofei-iatsenko** [comment]
I think it's a good idea. We would happily accept PR on this. I think it should be a separate setting (not related to attributes)

**Comment by timofei-iatsenko** [comment]
Opened a PR https://github.com/lingui/eslint-plugin/pull/20
